### PR TITLE
Added forgotten attribute for consistency

### PR DESCRIPTION
**Pull request by arjantijms**

### CODE REVIEW

File: src/main/java/javax/security/enterprise/authentication/mechanism/http/RememberMe.java [hunk 1]
```diff
@@ -195,6 +194,14 @@
...
+     * Flag to determine if remember me should be used.
+     * 
+     * @return Flag to determine if remember me should be used
+     */
+    @Nonbinding
+    boolean isRememberMe() default true;
+    
+    /**
      * EL expression to determine if remember me should be used. This is evaluated
      * for every request requiring authentication. The expression needs to evaluate
```
Comment (wmhopkins) [inline src/main/java/javax/security/enterprise/authentication/mechanism/http/RememberMe.java:206]: Shouldn't the javadoc for isRememberMeExpression be updated to reflect that both immediate and deferred expression syntax is allowed?

**Review by wmhopkins — CHANGES_REQUESTED** [review]
I'm OK with this, but see my comment about javadoc for the Expression flavor.

**Comment by arjantijms** [comment]
I think the class level documentation already captures that:

 * <p>

 * EL expressions in attributes of type <code>String</code> are evaluated
for every request requiring

 * authentication. Both immediate and deferred syntax is supported, but
effectively the semantics

 * are always deferred.

**Comment by wmhopkins** [comment]
OK.

**Pull request merged**